site/docs/components/calendar/usage.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the component
+
+#### When to use
+
+- To allow users to input a date or range of dates.
+
+#### When not to use
+
+- To display complex calendar information.
```

Comment:
This needs to be expanded on and explained.

Reply:
@ivan-calderon could would the recommendation be in that case? use it combined with another component that will display that info?

Reply:
'When not to use' case scenario could be:
- For calendar-related tasks other than selecting a date, such as picking a time or time-zone; or displaying event information. 
- If the date or range is pre-determined and can’t be specified by the user. Consider showing date options in a dropdown instead. (This comes from the UITK site)